velox/exec/tests/ExpandTest.cpp

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec::test;
+
+class ExpandTest : public OperatorTestBase {};
+
+TEST_F(ExpandTest, groupingSets) {
```

Comment:
@mbasmanova  Spark utilizes the expand operation in scenarios involving `grouping sets`, `cube`, `rollup`, and `count(distinct)`. I have already included tests for `grouping sets`, `cube`, and `rollup`. However, it seems that Velox does not support the `count(distinct)` expression, so I have not added tests for it.

Reply:
@JkSelf count(distinct) and similar are supported: https://github.com/facebookincubator/velox/pull/5632